### PR TITLE
Make the color of zero-contribution-squares in the activity heatmap more subtle

### PR DESCRIPTION
**Pull request by evur**
The previous color had a too high contrast with the background.

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by yardenshoham** [comment]
Before & after screenshots?

**Comment by delvh** [comment]
(Simply try it out on try.gitea.io and see for yourself…)

But yeah, screenshots are always nice to have.

**Comment by yardenshoham** [comment]
# Before
image
# After
image

**Comment by silverwind** [comment]
At least on my screen, this seems to low contrast on arc-green:

Screenshot 2023-05-16 at 21 28 14

Light theme looks alright:

Screenshot 2023-05-16 at 21 29 36

There is no easy solution to have same contrast on both themes with equal alpha shades as the contrast between secondary and background color is different on the themes. arc-green needs to be made darker I guess.

The difference is pretty minor. Mayb going to alpha 60 is also an option as a compromise.

**Comment by evur** [comment]
I chose the color with arc-green in mind. But I can also up it to `60` like @silverwind proposed.

**Comment by silverwind** [comment]
I thought you had optimized for ligh theme.

It does look better on @yardenshoham's screenshot than on mine, likely because of different OS color profiles.

File: web_src/js/components/ActivityHeatmap.vue [hunk 1]
```diff
@@ -31,8 +31,8 @@ export default {
   },
   data: () => ({
     colorRange: [
-      'var(--color-secondary-alpha-70)',
-      'var(--color-secondary-alpha-70)',
+      'var(--color-secondary-alpha-50)',
+      'var(--color-secondary-alpha-50)',
```
Comment (silverwind) [inline web_src/js/components/ActivityHeatmap.vue]:
```suggestion
      'var(--color-secondary-alpha-60)',
      'var(--color-secondary-alpha-60)',
```

**Comment by silverwind** [comment]
Changed it to 60 as discussed.

**Review by silverwind — APPROVED** [review]

**Pull request merged**